src/core_plugins/vega/public/vega_editor_template.html

```diff
@@ -0,0 +1,75 @@
+<div class="vegaEditor" ng-controller="VegaEditorController">
+  <div
+    ui-ace="{
+      onLoad: aceLoaded,
+      useWrapMode: true,
+      advanced: {
+        highlightActiveLine: false
+      },
+      rendererOptions: {
+        showPrintMargin: false
+      },
+      mode: 'hjson'
+    }"
+    id="vegaAceEditor"
+    ng-model="vis.params.spec"
+    data-test-subj="vega-editor"
+  ></div>
+
+  <div id="editor_actions">
+    <span dropdown>
+      <button
+        id="vegaHelp"
+        class="editor_action"
+        dropdown-toggle href="#"
+        aria-label="Vega help"
+      >
+        <span class="kuiIcon fa-question-circle"></span>
+      </button>
+      <ul
+        class="dropdown-menu"
+        role="menu"
+        aria-labelledby="vegaHelp"
+      >
+        <li role="menuitem">
+          <a target="_blank" href="https://www.elastic.co/guide/en/kibana/master/vega-graph.html">
```

Comment:
Add `rel="noopener noreferrer"` no all links, that open with `target="_blank"`... [always](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)!

Reply:
Thanks, I also found this one - https://github.com/elastic/kibana/pull/15948

Reply:
And these - i'm not as certain about them. https://github.com/elastic/kibana/pull/15949